sdk/monitor/azure-monitor-opentelemetry-exporter/azure/monitor/opentelemetry/exporter/export/_base.py

```diff
@@ -86,6 +91,11 @@ def __init__(self, **kwargs: Any) -> None:
                 name="{} Storage".format(self.__class__.__name__),
                 lease_period=self._storage_min_retry_interval,
             )
+        # statsbeat initialization
+        if self._check_stats_collection():
+            # Import here to avoid circular dependencies
+            from azure.monitor.opentelemetry.exporter.statsbeat._statsbeat import collect_statsbeat_metrics
```

Comment:
Could you explain the circular dependencies?

Reply:
`_base.py` depends on `_statsbeat.py` for `collect_statsbeat_metrics` and `statsbeat.py` depends on `_StatsBeatExporter` which inherits `BaseExporter` from `_base.py`.

Reply:
Because the StatsBeatExporter not only extends the base exporter, but the metrics exporter as well, which needs to collect statsbeat, there is no clear fix for this right now.